content/en/docs/tasks/access-kubernetes-api/configure-aggregation-layer.md

```diff
@@ -224,6 +224,55 @@ If you are not running kube-proxy on a host running the API server, then you mus
 
 {{% /capture %}}
 
+### Register APIService objects
```

Comment:
Wasn't sure what the best place to document kube-aggregator is. After some searching, this seemed like the best place, but happy to move if anyone knows of a more appropriate doc.